shell/platform/BUILD.gn

```diff
@@ -13,7 +13,7 @@ group("platform") {
     deps = [ "android" ]
   } else if (is_linux) {
     deps = []
-    if (enable_desktop_embeddings) {
+    if (enable_desktop_embeddings && !embedder_for_pc) {
       deps += [ "linux" ]
     }
     if (build_tizen_shell) {
```

Comment:
You don't need a new variable `embedder_for_pc`. There's already `enable_desktop_embeddings`.

```gn
if (build_tizen_shell) {
  deps += [ "tizen" ]
} else if (enable_desktop_embeddings) {
  deps += [ "linux" ]
}
```

Reply:
You're right! Thanks!